Product/Production/Services/DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/docsubmission/outbound/deferred/response/StandardOutboundDocSubmissionDeferredResponse.java

```diff
@@ -91,17 +90,17 @@ private RespondingGatewayProvideAndRegisterDocumentSetSecuredResponseRequestType
     }
 
     private boolean isPolicyValid(RespondingGatewayProvideAndRegisterDocumentSetSecuredResponseRequestType request,
-            AssertionType assertion) {
+        AssertionType assertion) {
         boolean isValid = false;
 
         if (hasNhinTargetHomeCommunityId(request)) {
             String senderHCID = getSubjectHelper().determineSendingHomeCommunityId(assertion.getHomeCommunity(),
-                    assertion);
+                assertion);
             String receiverHCID = getNhinTargetHomeCommunityId(request);
             String direction = NhincConstants.POLICYENGINE_OUTBOUND_DIRECTION;
 
```

Comment:
No need for declaring receiverHCID and direction.We can use inline method calls here.